docs/mmteb/points/636.jsonl

```diff
@@ -0,0 +1,2 @@
+{"GitHub": "jupyterjazz", "New dataset": 326}
```

Comment:
It's 2 points for a New dataset even with multiple languages. You may add 4 points per new language if it's not already handled by another task.

Reply:
How many points do you suggest I should add? 81 out of the 115 Belebele languages are not included in other retrieval datasets, that's why I added 2 + 81*4, I think this is what other PRs did as well

Reply:
@KennethEnevoldsen we need some help here with the points

Reply:
Ahh this actually already happened back in #583 and we have had similar cases before (though never on this scale). We might consider changing the new dataset scores to be a max of 50-100. Both in terms on value added to the benchmark and effort I believe this is reasonable. We will have to update previous scores though.

@jupyterjazz how would you feel about such a change? We know that the points system is imperfect (always will be) and we don't want to discourage contributors who have put many hours into review etc. - I hope that the solution can strike the right balance.

Just checking in with @isaac-chung (reviewer, and large contributor) and @davidstap (previous contributor of large dataset) here as well.

Reply:
Hey @KennethEnevoldsen, thanks for the response. I'm okay with capping the max score of bonuses, I think it's fair. 

Can you please also help us out in this discussion?
https://github.com/embeddings-benchmark/mteb/pull/636#discussion_r1596046376

Reply:
No issues with points cap for a single dataset 👍

Reply:
50 points seem fair, we'll open an issue to fix points of other PRs. @KennethEnevoldsen @isaac-chung do you agree?

Reply:
👍

Reply:
Saw the tag just now, thanks @KennethEnevoldsen. Not sure how I feel about changing the point system for previous submissions. I get that the change might better reflect effort that went into contributions. Changing the rules now (for future contributions) seems fair, since people can determine whether they participate given these new rules. Changing rules and applying new rules retroactively to previously earned points seems unfair to me.

Reply:
@KennethEnevoldsen @isaac-chung I proposed here a new rule for the pointing system as discussed above: https://github.com/embeddings-benchmark/mteb/pull/666. 

@davidstap Thank you for bringing that up. This change is by no means to diminish any contributions that are already made. We already warned contributors that the system may change in the following line: 
[> **Note**: The purpose of the point system is not to barrier collaboration, but to reward contributions. We might adjust the point requirement lower to accommodate more co-authorship if needed.](https://github.com/embeddings-benchmark/mteb/blob/376568330d79367f1330c3faf7ba3f6533cbd41f/docs/mmteb/readme.md?plain=1#L46)

Contributors will still be recognized as co-authors if they meet the criteria in the points guide.

Reply:
Thanks for the response @davidstap. The specific reason why I would like to reduce previous contributions as one dataset has obtained >3000 points for a single dataset contribution. We should have taken the discussion at the time, but there are many PRs and I understand that it wasn't caught.

Keeping the score as it is suggests to the contributors that the bibleNLP dataset is worth more than ALL the remaining contributions combined. I don't believe that is fair to the other contributors of this project so it is clearly something that we will have to change.

Again this should have been caught in review and I am sad that it wasn't. I hope we can find a solution that fits everyone.

Reply:
I'd like to offer my sincere apologies for missing this and not raising this earlier during the original review for the BibleNLP dataset 🙏

Reply:
> @davidstap Thank you for bringing that up. This change is by no means to diminish any contributions that are already made. We already warned contributors that the system may change in the following line:

@imenelydiaker Thanks for the pointer. I read that statement as: we might decide that a lower point _requirement_ (i.e., less than 10 points) merits authorship, not that the point _system_ might change.

Reply:
> Thanks for the response @davidstap. The specific reason why I would like to reduce previous contributions as one dataset has obtained >3000 points for a single dataset contribution. We should have taken the discussion at the time, but there are many PRs and I understand that it wasn't caught.
> 
> Keeping the score as it is suggests to the contributors that the bibleNLP dataset is worth more than ALL the remaining contributions combined. I don't believe that is fair to the other contributors of this project so it is clearly something that we will have to change.
> 
> Again this should have been caught in review and I am sad that it wasn't. I hope we can find a solution that fits everyone.

I get your concern about one dataset contribution racking up a crazy number of points. It seems unfair when that dataset is worth more than all other contributions combined.

OTOH, I think _massively multilingual_ datasets such as BibleNLP, but also FLORES and NTREX, are _extremely_ valuable for a _multilingual_ benchmark. They cover a ton of languages while keeping the domain and other confounders constant, which is crucial when one wants to test how well different languages perform in the same setting. When you're dealing with languages from different datasets, it's hard to compare performance because there are so many other confounding factors at play, like the topic/domain or length of the samples.


That’s why I think it’s much more valuable to have a single dataset with 500 languages, instead of 10 datasets with 50 distinct languages each, even if the set of languages is the same. The new point system (cap at 50/100 points) would not capture this at all. Of course no point system is perfect, but IMO this change would not be an improvement.

Reply:
> OTOH, I think _massively multilingual_ datasets such as BibleNLP, but also FLORES and NTREX, are _extremely_ valuable for a _multilingual_ benchmark. They cover a ton of languages while keeping the domain and other confounders constant, which is crucial when one wants to test how well different languages perform in the same setting. When you're dealing with languages from different datasets, it's hard to compare performance because there are so many other confounding factors at play, like the topic/domain or length of the samples.

BibleNLP, Flores and NTREX can only be used in one task for a multilingual benchmark, although it's be nice to remind that the key idea behind MTEB is the ability to evaluate a model accross different tasks and languages. These datasets are cross-lingual, most of them are parallel texts that were built using translation, so basically they contain the same information across all languages. 

Having distinct datasets with different languages is also valuable since we're not only focused on one style of writing or one source of data. It would clearly be unfair to reward a multilingual dataset more than other contributions like adding a new emmbedding evaluation task. 

This new pointing system will not diminish your contribution, you'll still get 100 points for your two datasets which will get you in the top list of co-authors which is already good. It will just make it fair towards other contributors that have also put efforts into this.

Again sorry for the confusion, we should have fixed this earlier. But we're humans and we make mistakes 🙂

Reply:
> That’s why I think it’s much more valuable to have a single dataset with 500 languages, instead of 10 datasets with 50 distinct languages each, even if the set of languages is the same. The new point system (cap at 50/100 points) would not capture this at all. Of course no point system is perfect, but IMO this change would not be an improvement.

of course, diversity of languages is not the only target of measure. Diversity of tasks is also important. E.g. A model performing well on bible datasets might perform well based on heuristics (e.g. does Jesus appear at the beginning of the sentence), but 50x datasets in 50 different languages cover different domains, tasks and sources and thus I would believe measures a broader spectrum (when we do a factor analysis of task x language scores I believe we will see the same thing as well).

With all of this being said. I would suggest that we set a limit to 50 points for the new datasets but raise the point limit to 100 points for already accepted PRs as well as ongoing PRs. I believe this strikes a fair balance between stating that multilingual datasets are extremely valuable while not overstating their value.

Reply:
@imenelydiaker @KennethEnevoldsen thanks for your additional comments. It seems I have overstated the importance of extremely multilingual datasets in the context of mmteb. I think this stems from my background in multilingual MT, where these kinds of datasets are very valuable. Thanks again for organizing this community effort ☺️

Reply:
Thanks for understanding @davidstap. Again we very much appreciate your contributions and this is by no means meant to understate it.